pkg/kubelet/kubelet.go

```diff
@@ -171,6 +173,9 @@ const (
 
 	// Minimum number of dead containers to keep in a pod
 	minDeadContainerInPod = 1
+
+	// Period for informers to resync
+	resyncPeriod = time.Minute
```

Comment:
why 1 minute? Can we document the reasoning behind this in the comment

Reply:
I saw this 1 minute being used at another place.  We probably don't need that.